### PR TITLE
Record delay between shader module creation and first use

### PR DESCRIPTION
**Pull request by kuhar**
... in pipeline creation.
    
The delay info is only logged to the even log.

### CODE REVIEW

**Comment by kuhar** [comment]
Only the second commit is part of this PR. The first one is submitted for review separately: https://github.com/googlestadia/performance-layers/pull/47.

File: layer/layer_data.h [hunk 1]
```diff
@@ -314,6 +314,9 @@ class LayerData {
   void LogEventOnly(std::string_view event_type,
                     std::string_view extra_content = "") const;
 
+  // Returns a string representation of |shader_hash|.
```
Comment (vettoreldaniele) [inline layer/layer_data.h]: Comment should say `|hash|`

**Review by vettoreldaniele — APPROVED** [review]
Thanks Jakub. I only reviewed the second commit as you mentioned.

**Comment by kuhar** [comment]
Thanks for catching the outdated comment, @vettoreldaniele. Could you also take a look at #47?

**Comment by kuhar** [comment]
V2: Rebased.

**Pull request merged**